cleanup.sh

```diff
@@ -13,7 +13,7 @@ fi
 set -euo pipefail
 
 log() {
-    echo "•" $*
+    echo "•" "$@"
 }
```

Comment:
I suggest removing this and putting `. lib/functions.sh` after we cd to SCRIPT_DIR on Line 5

Reply:
done